apps/dot-cdn/src/app/app.module.ts

```diff
@@ -17,7 +22,11 @@ import { HttpClientModule } from '@angular/common/http';
         DropdownModule,
         BrowserAnimationsModule,
         HttpClientModule,
-        RouterModule.forRoot([])
+        RouterModule.forRoot([]),
+        TabViewModule,
+        ChartModule,
+        InputTextareaModule,
+        ButtonModule
```

Comment:
I think` InputTextareaModule`, and `ButtonModule` are dups
https://github.com/dotCMS/core-web/blob/5a56251b636fc2d7a18b2d9da9f9b453eb6399d3/apps/dotcms-ui/src/app/modules.ts#L68

Reply:
Why? Sorry I don't understand. I'm importing this in the new dotCDN's app module

Reply:
https://github.com/dotCMS/core-web/pull/1650/files/578fbea47a9c5b609e998e029f84882bf10aa94e#r614367846